src/components/header-top/header-top.tsx

```diff
@@ -0,0 +1,13 @@
+import { AddMovieButton } from "../add-movie-button/add-movie-button";
+import { Logo } from "../logo/logo";
+
+import styles from './header-top.module.scss'
+
+export const HeaderTop =() => {
+  return (
```

Comment:
For functional components I advise you to definitely use implicit `return` statement with feature of arrow functions (AF)